Source/UI/iOS/ECLogManagerIOSUISupport.m

```diff
@@ -65,25 +65,25 @@ - (void)logManagerDidStartup:(ECLogManager*)manager
 #endif
 }
 
-- (void)installGestureRecognizer
-{
-	UIWindow* window = [UIApplication sharedApplication].windows[0];
-	UILongPressGestureRecognizer* recognizer = [[Test alloc] initWithTarget:self action:@selector(showUI)];
-	recognizer.numberOfTouchesRequired = 2;
-	[window addGestureRecognizer:recognizer];
+- (void)logManagerWillShowUI:(ECLogManager *)manager {
+	[self showUI];
 }
-
-+ (void)showUI
+	
+- (void)installGestureRecognizer
 {
-	[[self sharedInstance] showUI];
+	UIWindow* window = UIApplication.sharedApplication.keyWindow;
+	if (window) {
+		UILongPressGestureRecognizer* recognizer = [[Test alloc] initWithTarget:self action:@selector(showUI)];
```

Comment:
Looks like this may be historical, but it would probably worth tidying this up. (I.e removing the `Test` class and just calling `[UILongPressGestureRecognizer alloc]`).